icons/user.tsx

```diff
@@ -0,0 +1,71 @@
+'use client';
+
+import type { Variants } from 'framer-motion';
+import { motion, useAnimation } from 'framer-motion';
+
+const pathVariant: Variants = {
+  normal: { pathLength: 1, opacity: 1, pathOffset: 0 },
+  animate: {
+    pathLength: [0, 1],
+    opacity: [0, 1],
+    pathOffset: [1, 0],
+  },
+};
+
+const circleVariant: Variants = {
+  normal: {
+    pathLength: 1,
+    opacity: 1,
```

Comment:
i guess you don't need `opacity` here cause you don't change it

Reply:
Opacity Removed